doc/user_guide/user_guide.md

```diff
@@ -167,6 +251,7 @@ exaslc-ci export-and-scan-vulnerabilities \
 - `--docker-user TEXT`
 - `--docker-password TEXT`
 - `--github-output-var TEXT`
+- `--release/--no-release`: If True, the "docker_release_repository" entry of `build_config.json` will be used for uploading the Script-Languages container image. Otherwise, the "docker_release_repository" will be used.
```

Comment:
```suggestion
- `--release`: If specified, then run `export-and-scan-vulnerabilities` in mode _release_, always re-building all Docker images and publishing the images to "docker_release_repository". Otherwise only rebuild outdated Docker images and publish them to "docker_build_repository".
```